airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -244,7 +244,7 @@ def read_file(self, container_name: str, blob_name: str, **kwargs):
         :param kwargs: Optional keyword arguments that `BlobClient.download_blob` takes.
         :type kwargs: object
         """
-        return self.download(container_name, blob_name, **kwargs).readall()
+        return self.download(container_name, blob_name, **kwargs).content_as_text()
```

Comment:
```suggestion
        return self.download(container_name, blob_name, **kwargs).content_as_text(max_concurrency=1, encoding='utf-8')
```

Reply:
I check the Microsoft document you gave above and find these two parameters are default value.
Ok, I can add this as you suggested

Reply:
Hey, seen it. Please don't add

Reply:
You have to rebase though

Reply:
I have rabased follow the Pull Request Guidelines, please check it